Assets/Scripts/GameManager.cs

```diff
@@ -61,6 +63,7 @@ IEnumerator WaitForMission(MissionButton missionButton)
         playerData.playerMoney += mission.missionValue;
         missionButton.ResetMissionTime();
         playerMoneyText.text = "$" + playerData.playerMoney;
+        mission.inProgress = false; 
```

Comment:
it probably makes more sense to lower the  `yield return new WaitForSeconds(1);` to something like 0.1 seconds so the slider can move smoothly, in case there's a timer that goes less than a second once they have upgrades